packages/generators/app/src/resources/files/ts/src/admin/app.example.tsx

```diff
@@ -29,7 +29,7 @@ export default {
       // 'zh',
     ],
   },
-  bootstrap(app) {
+  bootstrap(app: unknown) {
```

Comment:
Isn't this the front-end Strapi app object? I think there's a type for it somewhere

Reply:
@joshuaellis does it ring a bell?

Reply:
There's not one exported, but actually, why dont we do that now? 

if we add:

```ts
export type { StrapiApp } from './StrapiApp';	
```

to the strapi-admin package then you can import it via:

```ts
export type { StrapiApp } from '@strapi/strapi/admin'
```

Reply:
Then it'll be typed!

Reply:
Done, thanks :magic_wand:

Reply:
beautiful!